dist/docker/debian/build_docker.sh

```diff
@@ -102,6 +109,10 @@ bconfig --entrypoint  '["/docker-entrypoint.py"]'
 bconfig --cmd  ''
 bconfig --port 10000 --port 9042 --port 9160 --port 9180 --port 7000 --port 7001 --port 22
 bconfig --volume "/var/lib/scylla"
+bconfig --label org.opencontainers.image.ref.name
+bconfig --label org.opencontainers.image.version
```

Comment:
What does this do?

Reply:
It removes those default ones.

Reply:
Add an empty string, the default is 
```
"org.opencontainers.image.ref.name": "ubuntu",
"org.opencontainers.image.version": "22.04",
```
@mykaul asked to remove those

Reply:
shouldn't we put our own instead?

Reply:
We should.

Reply:
What should we have instead?

Reply:
Version is easy - we know what the version is (I'd add the complete version, inc. build ID, etc.

Reply:
Name could be just 'ScyllaDB'

Reply:
```
          "Labels": {
               "description": "ScyllaDB Open Source",
               "io.buildah.version": "1.33.2",
               "org.opencontainers.image.ref.name": "ScyllaDB",
               "org.opencontainers.image.version": "5.5.0~dev-0.20231221.3b5ae5101b23",
               "summary": "NoSQL data store using the seastar framework, compatible with Apache Cassandra"
          },
```

Changed